kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java

```diff
@@ -124,16 +121,12 @@ private EsSearchResponse doSearch(EsSearchRequest request) {
 
     try {
       HitsMetadata hits = getHits(searchResult);
-      Map<String, AggregationResponse> aggregations =
-          buildLegacyAggregationResponse(
-              request.getAggregations(), searchResult.getInternalAggregations());
-
       return new EsSearchResponse.Builder()
           .hits(hits)
-          .aggregations(aggregations)
+          .aggregations(parseAggregations(searchResult.getInternalAggregations()))
           .took(Duration.of(searchResult.getTookMicros(), ChronoUnit.MICROS).toMillis())
           .shardsMetadata(searchResult.getTotalNodes(), searchResult.getFailedNodes())
-          .debugMetadata(getDebugAggregations(searchResult.getInternalAggregations()))
+          .debugMetadata(Map.of())
```

Comment:
for now, we don't need this `debugMetadata` ?

Reply:
Correct, the previous use of `getDebugAggregations` was for rendering the raw response from the LogSearcherImpl. Since we directly return this now, there's not currently a need for returning debug metadata.

I left it initialized as an empty map so we remember we can use this, but this entire service will be revisited as part of the query layer rewrite.